editor/src/components/canvas/ui-jsx-canvas-renderer/ui-jsx-canvas-contexts.tsx

```diff
@@ -8,19 +8,18 @@ import { ProjectContentTreeRoot } from '../../assets'
 import type { TransientFileState, UIFileBase64Blobs } from '../../editor/store/editor-state'
 
 export interface MutableUtopiaContextProps {
-  requireResult: MapLike<any>
-  fileBlobs: UIFileBase64Blobs
-  rootScope: MapLike<any>
-  jsxFactoryFunctionName: string | null
+  [filePath: string]: {
+    mutableContext: {
```

Comment:
note: the reason we has this funky object shape is that requireResult and rootScope are effectively `any`s. We originally refactored the code to this: 

```typescript
export interface MutableUtopiaContextProps {
   [filePath: string]: {
     requireResult: MapLike<any>
     fileBlobs: UIFileBase64Blobs
     rootScope: MapLike<any>
     jsxFactoryFunctionName: string | null
  }
}
```

but since the places we use this object looked like this: `mutableContext.requireResult`, these places would not break the TS build, because mutableContext.requireResult is inferred to be of the shape 
```
{
     requireResult: MapLike<any>
     fileBlobs: UIFileBase64Blobs
     rootScope: MapLike<any>
     jsxFactoryFunctionName: string | null
  }
```
which can be passed into an `any`. So we wrapped the whole thing in yet another object with a single key, `mutableContext` to break out of `any` prison and let Typescript help us find all the places that use mutableContext in the old way